src/dagmc.cpp

```diff
@@ -484,33 +509,33 @@ void DAGUniverse::legacy_assign_material(
 
 void DAGUniverse::read_uwuw_materials()
 {
+  // If no filename was provided, disable read UWUW materials
+  uwuw_disabled = (filename_ == "");
 
-  int32_t next_material_id = 0;
-  for (const auto& m : model::materials) {
-    next_material_id = std::max(m->id_, next_material_id);
-  }
-  next_material_id++;
-
+  if (uwuw_disabled)
+    return;
```

Comment:
This variable name threw me for a loop. I thought it meant that UWUW was intentionally being disabled as an option. That might be a nice feature to add, but I don't think that was the intention here. Please correct me if I'm wrong. 

If you agree to my proposed change to `uses_uwuw` below, I think this state variable can go away and these lines can become:


```cpp
if (filename_ == "") return;
```

If you think that having the ability to ignore/disable UWUW materials as an option on the class could be useful, then we'll of course need to keep that state variable and set it in the constructor. Then we'd add an additional check above this one: 

```cpp
if (uwuw_disabled_) return;
```

Reply:
Your confusion is because originally I thought I did need the ability to disable (i.e. have UWUW exist but not used) with a corresponding method  but changed how I did things. I am very happy with your proposed change.

Reply:
done